external/fv3fit/fv3fit/_shared/config.py

```diff
@@ -356,7 +341,7 @@ def load_configs(
     output_data_path: str,
     timesteps_file=None,
     validation_timesteps_file=None,
-) -> Tuple[TrainingConfig, DataConfig, Optional[DataConfig]]:
+) -> Tuple[TrainingConfig, loaders.BatchesConfig, Optional[loaders.BatchesConfig]]:
```

Comment:
Note this is using the more restrictive type because it's analogous to what the old training config does, this function will be deleted completely when we can remove the old yaml format.